.changes/v3.13.0/1278-notes.md

```diff
@@ -0,0 +1 @@
+* Amend the test `TestAccVcdRdeDuplicate` so it doesn't fail on VCD 10.6+ [GH-1278]
```

Comment:
Would be good to have a shortened version of the explanation sentence of what changed in changelog too.
https://github.com/vmware/terraform-provider-vcd/pull/1278/files#diff-8f002150375f34b6c8946fb0e4abf627220b05a0a9da0f9e96dc8f9c9e047f4aR49

Reply:
Done, used the same explanation in both places, rephrased/simplified.